### PR TITLE
chore(ci): introduce github workflows

### PR DESCRIPTION
**Pull request by kduret**
## Description

introduce github workflows in order to package and deliver stream connectors (and libs)

**Fixes** MON-16173

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

### CODE REVIEW

File: .github/docker/Dockerfile.packaging-alma8 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:8
+
+RUN <<EOF
+
+dnf install -y rpm-build rpm-sign zstd
+dnf clean all
+
+EOF
```
Comment (paloth) [inline .github/docker/Dockerfile.packaging-alma8:10]: Why EOF ? Maybe the following code would be better 
```
RUN dnf install -y rpm-build rpm-sign zstd && \
    dnf clean all
```

File: .github/docker/Dockerfile.packaging-alma9 [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:9
+
+RUN <<EOF
+
+dnf install -y rpm-build rpm-sign zstd
+dnf clean all
+
+EOF
```
Comment (paloth) [inline .github/docker/Dockerfile.packaging-alma9:10]: Same as .github/docker/Dockerfile.packaging-alma8

File: .github/docker/Dockerfile.packaging-bullseye [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+FROM ${REGISTRY_URL}/debian:bullseye
+
+RUN <<EOF
+
+apt-get update
+apt-get install -y debmake pbuilder aptitude zstd
+apt-get clean
+
+EOF
```
Comment (paloth) [inline .github/docker/Dockerfile.packaging-bullseye:11]: See .github/docker/Dockerfile.packaging-alma8 review

**Review by sc979 — APPROVED** [review]

File: .github/docker/Dockerfile.packaging-alma8 (hunk 1 above)
Comment (kduret) [reply .github/docker/Dockerfile.packaging-alma8:10]: EOF makes instructions cleaner and more readable : https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/

**Review by paloth — APPROVED** [review]

**Pull request merged**